src/index.html

```diff
@@ -11,7 +11,108 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="formgroup">
+        <legend>Personal information</legend>
+        <label for="surname" class="forminput">Surname:
```

Comment:
if you wrap your input in label then you don't need  for attribute
```suggestion
        <label for="surname" class="forminput">
          Surname:
```